### PR TITLE
feat: added responsive images

### PR DESCRIPTION
**Pull request by varnita619**
- added responsive image
- added round image

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Everything is looking great.

Only The round image is not visible because the image has a background white. You can change the image so that it round can be visible.

**Review by jeetsdev — APPROVED** [review]
LGTM 💯

**Review by vinc3nati — APPROVED** [review]
Good to go!

File: Components/images/images.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    border-radius: 10px;
+    border: 2px solid var(--light-border-color);
+    background-color: var(--background-color);
+    height: auto;
+    width: 100%;
+}
+
+.responsive{
+    width: 100%;
+    max-width: 400px;
```
Comment (manish591) [inline Components/images/images.css:19]: ### Don't add max-width on the image itself
What if your component library user wanted the image to grow more than 400px?
You should give max width 100% to the image and if you want to restrict this image from growing then add max-width: 400px on the image container.

**Comment by varnita619** [comment]
> Everything is looking great.
> 
> Only The round image is not visible because the image has a background white. You can change the image so that it round can be visible.

sure

**Review by manish591 — COMMENTED** [review]
Your code looking clean.

**Pull request merged**